extensions/linear/src/components/ProjectIssues.tsx

```diff
@@ -18,9 +22,54 @@ type ProjectIssuesProps = {
 
 export default function ProjectIssues({ projectId, priorities, me, users }: ProjectIssuesProps) {
   const { issues, isLoadingIssues, mutateList } = useIssues(getProjectIssues, [projectId]);
+  const [milestone, setMilestone] = useCachedState<string>("");
```

Comment:
Can't you use `storeValue` on `List.Dropdown` instead?

Reply:
Good question. I can try to make it work. Would that make a big difference?

Reply:
The reason why I'm using it on the cache is so I can use for filtering the list.

Reply:
It doesn't make a big difference, it's just a bit more maintainable.